steam-buddy

```diff
@@ -134,8 +134,7 @@ def edit(platform, name):
 			return template('flathub_edit', app=application, platform="flathub", platformName="Flathub",
 							name=name, )
 		else:
-			# TODO: This should probably be changed to a 404
-			redirect('/platforms/flathub')
+			abort(404, '{} wasn\'t found in the Flathub API'.format(name))
```

Comment:
here too?

Reply:
My bad, I overlooked that one. Fixed now